### ISSUE TITLE
Add test `groups` to ActiveData

### ISSUE THREAD

**Issue by ahal** [klahnakoski/TestLog-ETL#61]
The mozlog spec says that `suite_start` actions contain a `tests` key that can optionally be keyed by `group`. These `group`s are typically the manifest and would be very valuable information to have in ActiveData. See:
https://firefox-source-docs.mozilla.org/mozbase/mozlog.html#data-format

It looks like we currently ignore this `tests` key completely:
https://github.com/klahnakoski/ActiveData-ETL/blob/ffcf6499ca50b1bc2c7b276ccac84877189baa35/activedata_etl/transforms/unittest_logs_to_sink.py#L215

(presumably because we get the test names from the `test_start` action instead).

We should create:

1. `run.groups` (or maybe `run.suite.groups`?) which will contain the set of manifests that ran in the suite.
2. `result.group` which will contain the manifest that contains a given test.

Typically the `tests` key will look something like:

```
{
  "path/to/manifestA.ini": ["path/to/testA", "path/to/testB", ...],
  "path/to/manifestB.ini": [ ... ],
  ...
}
```

Though if the test suite doesn't provide manifest information, then it will just say:

```
{
  "default": ["path/to/testA", "path/to/testB", ...]
}
```

In this case, both `run.groups` and `result.group` should just be `Null`.

**Comment by marco-c** [comment]
What's missing now that #63 was merged?

**Comment by marco-c** [comment]
Another question: could we backfill the data for the past few months (9 would be best)? So I can experiment with using this data in the ML model for test selection (otherwise I have to wait a few months to have enough training data).

**Comment by klahnakoski** [comment]
@marco-c  what's missing is some confirmation you can work with the new data.

@marco-c 9 months is too much!  I do not think the tc artifacts are kept that long. ActiveData only stores 6-7 weeks of data, but the S3 buckets go back 2 years.  Maybe you are suggesting we fill the S3 buckets back as far as we can, and you will use those?

**Comment by marco-c** [comment]
> @marco-c what's missing is some confirmation you can work with the new data.

I've checked using mozci, and it is looking good to me.
The only two problems I saw:
- web-platform-tests are using "default", but IIRC that's a known shortcoming;
- when a task contains a single manifest, e.g. Oc3vnkIFTq-G9pL4jsDc1g, groups was a string instead of a list of strings.

I'm not sure if the latter is a problem in ActiveData or in mozci.


> @marco-c 9 months is too much! I do not think the tc artifacts are kept that long. ActiveData only stores 6-7 weeks of data, but the S3 buckets go back 2 years. Maybe you are suggesting we fill the S3 buckets back as far as we can, and you will use those?

The longer the better, but we can also just wait for new data to come in and I will use it once we have enough. So, it depends on how costly it is to do this backfilling.
For the current model, which is only using tables which hold 2 years of data, I'm using 9 months (6 months for the actual training, the previous 3 for calculating failure history of all tasks).
